src/seedu/addressbook/AddressBook.java

```diff
@@ -277,12 +282,10 @@ private static void processProgramArgs(String[] args) {
      * Exits if the file name is not acceptable.
      */
     private static void setupGivenFileForStorage(String filePath) {
-
         if (!isValidFilePath(filePath)) {
-            showToUser(String.format(MESSAGE_INVALID_FILE, filePath));
+			showToUser(new String[] { String.format(MESSAGE_INVALID_FILE, filePath) });
```

Comment:
Indentation problem here?. Our coding standard requires you to use 4 spaces instead of tabs. You can configure Eclipse to convert tabs to spaces automatically.